kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/FallbackableTypeFactory.java

```diff
@@ -23,6 +23,7 @@
 import com.fasterxml.jackson.databind.util.ArrayBuilders;
 import com.fasterxml.jackson.databind.util.ClassUtil;
 import com.fasterxml.jackson.databind.util.LRUMap;
```

Comment:
Shouldn't this line be removed now?

Reply:
Hi @jesuino  , you can use this diff for FallbackableTypeFactory to align with jackson 2.12. (Note that this was not a PR to kiegroup but I wrote this to mbiarnes' repo when he was working on the wildfly update)

https://github.com/mbiarnes/droolsjbpm-integration/pull/8/files#diff-a5cbc3c1d03d90fca3a485f7bf685cee697c05878c17dea8969e19f9b1175c61

Of course, I can apply it later after you finish the current big PR. Thanks!

Reply:
Hello @tkobayas 

Sorry I didn't see your changes! We did faced it and solved it in a slightly different way in the revapi plugin.

When trying to apply the diff I had conflicts, would you please review the changes from this PR and possibly send a PR to my branch?

Thanks!

Reply:
@jesuino Sent a PR : https://github.com/jesuino/droolsjbpm-integration/pull/12 Thanks!